lib/telemetry_metrics_statsd/formatter.ex

```diff
@@ -1,35 +1,12 @@
 defmodule TelemetryMetricsStatsd.Formatter do
   @moduledoc false
```

Comment:
We're gonna need documentation for this module :)

Reply:
I thought about this. Do you want to open it to public for extension at day 1? It could potentially be left private for a bit as it solidifies and more formatters are added.

Reply:
Makes sense!